usmqe_tests/api/tendrl/test_volume.py

```diff
@@ -0,0 +1,271 @@
+
+        Get list of attributes needed to use in cluster volume creation with given cluster_id.
+
+        .. test_step:: 1
+
+                Connect to Tendrl API via POST request to ``APIURL/:cluster_id/GlusterCreateVolume``
+                Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **202** with data ``{"message": "Accepted"}``.
+                """
+    api = tendrlapi.ApiGluster()
+
+    volume_data = {
+        "Volume.volname": pytest.config.getini("usm_volume_name"),
+        "Volume.bricks": valid_volume_bricks
+    }
+
+    job_id = api.create_volume(valid_cluster_id, volume_data)["job_id"]
+    api.wait_for_job_status(job_id)
+    """@pylatest api/gluster.create_volume
+        API-gluster: create_volume
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
+
+        Description
+        ===========
+
+        Check if there is created volume on gluster nodes via CLI.
+
+        .. test_step:: 2
+
+            Connect to gluster node machine via ssh and run
+            ``gluster volume info command``
+
+        .. test_result:: 2
+
+            There should be listed gluster volume named ``Vol_test``.
+
+            """
+    storage = gluster.GlusterCommon()
+    storage.find_volume_name(pytest.config.getini("usm_volume_name"))
+
+    # TODO change
+    xml = storage.run_on_node(command="volume info")
+    volume_id = xml.findtext("./volInfo/volumes/volume/id")
+    value = api.get_volume_attribute(valid_cluster_id, volume_id, "name")
+    pytest.check(value == pytest.config.getini("usm_volume_name"))
+
+
+def test_create_volume_invalid(invalid_cluster_id, invalid_volume_name, invalid_volume_bricks):
+    """@pylatest api/gluster.create_volume
+        API-gluster: create_volume
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
+
+        Description
+        ===========
+
+        Get list of attributes needed to use in cluster volume creation with given cluster_id.
+
+        .. test_step:: 1
+
+                Connect to Tendrl API via POST request to ``APIURL/:cluster_id/GlusterCreateVolume``
+                Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **202** with data ``{"message": "Accepted"}``.
+                """
+    api = tendrlapi.ApiGluster()
+
+    volume_data = {
+        "Volume.volname": invalid_volume_name,
+        "Volume.bricks": invalid_volume_bricks,
+    }
+
+    job_id = api.create_volume(invalid_cluster_id, volume_data)["job_id"]
+    api.wait_for_job_status(
+            job_id,
+            status="failed",
+            issue="https://github.com/Tendrl/tendrl-api/issues/33")
+    # TODO check correctly server response or etcd job status
```

Comment:
Yes, I expect that status will be `failed` but there is no error status implemented.